lib/listen/listener.rb

```diff
@@ -82,42 +79,39 @@ def initialize(*dirs, &block)
       super() # FSM
     end
 
-    default_state :initializing
+    start_state :initializing
 
     state :initializing, to: [:backend_started, :stopped]
 
-    state :backend_started, to: [:frontend_ready, :stopped] do
+    state :backend_started, to: [:processing_events, :stopped] do
```

Comment:
Someday I'll get rid of `backend_started` and fold it into `proceessing_events`... which I'll rename to `started`.